### ISSUE TITLE
Run some code once ORM is loaded

### ISSUE THREAD

**Issue by sirsavary** [balderdashy/sails#2919]
I was wondering how I could call a method / run some code once the ORM hook is loaded. I can't seem to find anything about doing this in the docs.

**Comment by tjwebb** [comment]
You can listen for the `hook:orm:loaded` event.

Here's an example: https://github.com/tjwebb/sails-permissions/blob/master/api/hooks/sails-permissions.js#L31

**Comment by sailsbot** [comment]
Thanks for posting, @sirsavary.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by tjwebb** [comment]
@sirsavary Thanks for using them and providing feedback! FYI: https://github.com/tjwebb/sails-permissions/issues/200
